### PR TITLE
Add check for isHighContrast before page load

### PR DESCRIPTION
**Pull request by jsmith176**

### CODE REVIEW

File: js/contraster.js [hunk 1]
```diff
@@ -16,6 +16,11 @@ bcpl.contraster = (($, browserStorage) => {
 	const contrasterSettings = {};
 
 	const localStorageHighContrastKey = 'isHighContrast';
+	const isHighContrast = localStorage.isHighContrast === 'true';
```
Comment (martypowell) [inline js/contraster.js]: if we have the key above, you should probably make this line:

```javascript
const isHighContrast = localStorage[localStorageHighContrastKey ] === 'true';
```

Comment (jsmith176) [reply js/contraster.js]: Okay, I'll make that change

Comment (martypowell) [reply js/contraster.js]: I think maybe you should be using `localStorage.getItem(key) ?`

Comment (jsmith176) [reply js/contraster.js]: Yeah, you are right.

Comment (jsmith176) [reply js/contraster.js]: Done

**Comment by martypowell** [comment]
@jsmith176 it looks like one test pass failed in travis, but i reran it and it seems ok, so go ahead and merge. Just take note of this in the future.

**Pull request merged**